controller.java

```diff
@@ -5,10 +5,10 @@ public static void main(String[] args) {
         int num1 = 10;
         int num2 = 20;
 
-        // Perform addition
-        int sum = num1 + num2;
-
+int summation = math.add(number2, num1);
+        int summation = math.add(num2, num1);
+        System.out.println("xofsz");
         // Print the result
-        System.out.println("The sum of " + num1 + " and " + num2 + " is: " + sum);
+        System.out.println("The result of " + num1 + " and " + num2 + " is: " + sum);
```

Comment:
change variable name to add
```suggestion
        System.out.println("The sum of " + num1 + " and " + num2 + " is: " + add);
```